app/components/accordion_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="usa-accordion usa-accordion--bordered">
+<%= tag.div(**tag_options, class: css_class) do %>
```

Comment:
I'd expect `**` to go last? Or is this `class:` overriding the `tag_options`?

Reply:
> I'd expect `**` to go last? Or is this `class:` overriding the `tag_options`?

Oops, I forgot about this comment. But yes, the `class:` needs to come after, since otherwise the custom class would override the `usa-accordion` base classes.